CHANGES.md

```diff
@@ -22,4 +23,4 @@
  - Optimize binarization.  [View pull request](https://github.com/ivadomed/ivadomed/pull/364)
  - Automate training improvement.  [View pull request](https://github.com/ivadomed/ivadomed/pull/362)
  - Complement to PR 342.  [View pull request](https://github.com/ivadomed/ivadomed/pull/361)
```

Comment:
can we update the title of this PR to be more explicit

Reply:
Done!

Reply:
Ah wrong PR! Im on it

Reply:
Ok done :)!